### DISCUSSION TITLE
Work on obsidian mobile 🙏

### DISCUSSION THREAD

**Discussion by gimsall** [mcndt/obsidian-toggl-integration#147] [Feature Requests]
I saw this amazing plug-in and thought it would be nice to add it on my home canvas in my tab...so i tried a trick i read it by TfT hacker ,he was talking about changing the manifest.json file ,with this plug-in it didn't work

**Comment by leoccyao** [comment]
See #66 for troubleshooting attempts so far; would be great to see more progress looking into getting mobile support working!
